test/string-class-test.js

```diff
@@ -0,0 +1,169 @@
+var expect = require('chai').expect;
```

Comment:
### Critical

Use of ES5 syntax `var`. This particular issue was called out during the defense session, why is it still here?